### ISSUE TITLE
I am unable to initialize GoogleSignInClientProxy in Android

### ISSUE THREAD

**Issue by Dev-GOM** [muzhigg/com.izhguzin.google-identity#1]
I followed everything but I am unable to initialize GoogleSignInClientProxy in Android.  Any examples of how to add a dependency to initialize it arbitrarily?

**Comment by muzhigg** [comment]
> I followed everything but I am unable to initialize GoogleSignInClientProxy in Android. Any examples of how to add a dependency to initialize it arbitrarily?

Are there any errors in the logcat?

**Comment by Dev-GOM** [comment]
> > I followed everything but I am unable to initialize GoogleSignInClientProxy in Android. Any examples of how to add a dependency to initialize it arbitrarily?
> 
> Are there any errors in the logcat?

![Screenshot_20230620_020826_metafellow](https://github.com/muzhigg/com.izhguzin.google-identity/assets/107686415/0aca3462-15c4-4206-883e-de8d47baefe6)

here it is

**Comment by muzhigg** [comment]
Strange, but I can't reproduce this error by following the instructions.

So, I need more information. It would be great if you could provide a link to your project repository.

If that's not possible, I need to know the version of Unity you're using, and please send a screenshot of the Android settings in the "Other" section.

**Comment by Dev-GOM** [comment]
> Strange, but I can't reproduce this error by following the instructions.
> 
> So, I need more information. It would be great if you could provide a link to your project repository.
> 
> If that's not possible, I need to know the version of Unity you're using, and please send a screenshot of the Android settings in the "Other" section.

As a result of checking, I unchecked the release of minify and it works normally.  It seems that the class could not be found due to obfuscation.  How can I use realease?

**Comment by muzhigg** [comment]
Yes, I have already figured out what the problem was. I will provide an update with the solution today.

Also, please let me know which version of Unity you are using. Because on version 2022.3.2f, the application crashes during initialization.

**Comment by Dev-GOM** [comment]
Oh my.. 2022.3.2f1 is in use.

> Yes, I have already figured out what the problem was. I will provide an update with the solution today.
> 
> Also, please let me know which version of Unity you are using. Because on version 2022.3.2f, the application crashes during initialization.

Oh my.. 2022.3.2f1 is in use.

**Comment by muzhigg** [comment]
So, I was unable to reproduce this error even with the "minify release" checkbox checked. I made some changes to the proguard-rules in the Android library. So, you can try to see if this error occurs again.
Here's what you need to do:
1. Remove the Google Identity package from the Package Manager.
2. Install the Google Identity package from the git URL "https://github.com/muzhigg/com.izhguzin.google-identity.git#class-not-found-fix".
3. Enable the "Minify Release" flag.

**Comment by Dev-GOM** [comment]
> So, I was unable to reproduce this error even with the "minify release" checkbox checked. I made some changes to the proguard-rules in the Android library. So, you can try to see if this error occurs again. Here's what you need to do:
> 
> 1. Remove the Google Identity package from the Package Manager.
> 2. Install the Google Identity package from the git URL "https://github.com/muzhigg/com.izhguzin.google-identity.git#class-not-found-fix".
> 3. Enable the "Minify Release" flag.

Yes, I'm processing another issue right now, so I'll do it as soon as it's finished. Thank you for your concern. :)
